modules/github-oidc-role/main.tf

```
@@ -0,0 +1,54 @@
+locals {
+  enabled             = module.this.enabled
+  aws_canned_policies = [for arn in var.aws_iam_policies : arn if can(regex("^arn:aws:iam::aws:policy/", arn))]
+  aws_policies        = length(local.aws_canned_policies) > 0 ? local.aws_canned_policies : null
+  policy_document_map = {
+    "gitops"        = local.gitops_policy
+    "lambda_cicd"   = local.lambda_cicd_policy
+    "inline_policy" = one(module.iam_policy.*.json)
+  }
+  active_policy_map = { for k, v in local.policy_document_map : k => v if v != null }
+}
+
+module "iam_policy" {
+  count = local.enabled && length(var.iam_policy) > 0 ? 1 : 0
```

since this is a cloud posse module, can we use `enabled` instead of `count`?

```suggestion
  enabled = local.enabled && length(var.iam_policy) > 0
```